ouroboros-network-framework/test/Test/Ouroboros/Network/Server2.hs

```diff
@@ -1092,7 +1094,7 @@ newtype MultiNodeScript req peerAddr = MultiNodeScript [ConnectionEvent req peer
 
 -- | A sequence of connection events that make up a test scenario for `prop_multinode_Sim_Pruning`.
 -- This test optimizes for triggering prunings.
-newtype MultiNodeScriptPruning req = MultiNodeScriptPruning [ConnectionEvent req TestAddr]
+data MultiNodeScriptPruning req = MultiNodeScriptPruning (Small Word32) [ConnectionEvent req TestAddr]
```

Comment:
Yes it is the hard limit and I wanted the generator to generate small values (close to 0) instead of truly random `Word32` values. This will make it easier to trigger pruning since the `MultiNodeScript` generator does not generate millions of connections